### PR TITLE
mail adapt debug manager.include

### PR DESCRIPTION
**Pull request by msh-odoo**
add item 'Manage Messages' in debug manager

### CODE REVIEW

File: addons/mail/static/src/js/tools/debug_manager.js [hunk 1]
```diff
@@ -1,33 +1,45 @@
...
-    },
+patch(ViewAdapter.prototype, 'mail.view_adapter', {
+    setup() {
+        const envWowl = this.env;
+        useDebugManager((accessRights) =>
+            setupDebugMailMessage(envWowl, this, this.props.viewParams.action)
+        );
+        this._super(...arguments);
+    }
 });
```
Comment (SimonGenin) [inline addons/mail/static/src/js/tools/debug_manager.js:15]: Don't patch to add elements to the debug manager. And don't do useDebugManager as a way in. 
Use the simpler `import { debugRegistry } from "@web/debug/debug_registry";`
```js
debugRegistry.add("mail.manage_messages", debugMailMessage);
```

Comment (msh-odoo) [reply addons/mail/static/src/js/tools/debug_manager.js:15]: @SimonGenin I wanted to call ```getSelectedIds``` here, do we have a alternative to call it any other way?

Comment (SimonGenin) [reply addons/mail/static/src/js/tools/debug_manager.js:15]: I'll ask my team tomorrow

Comment (SimonGenin) [reply addons/mail/static/src/js/tools/debug_manager.js:15]: @msh-odoo So we indeed didn't have a nice way around this. So in a way, your patch was one of the possible solution. We are trying to give the debugmanager a second registry. One for global items and one for more specific ones. 
You can see a wip here https://github.com/odoo-dev/odoo/pull/738

**Comment by aab-odoo** [comment]
@msh-odoo work done here https://github.com/odoo-dev/odoo/pull/880  on the current state of the branch

**Pull request closed without merging**